src/cmd-buildextend-installer

```diff
@@ -15,6 +15,11 @@ import tempfile
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import run_verbose, write_json, sha256sum_file
 from cmdlib import import_ostree_commit, Builds
+from pylorax.ltmpl import LoraxTemplateRunner
```

Comment:
As noted I think sharing code with Anaconda/lorax is a good idea, but we also now become a lot more coupled to it, and a downside of Python is that as a dynamic language it's harder to maintain ABI/stability (IMO).  If we go this route let's make sure the lorax maintainers understand they have another downstream consumer.

This dependency also currently only exists on s390x, but we do this import on all architectures.

I think that's probably fine; just noting.

Reply:
The biggest use of lorax in here is : `LoraxTemplateRunner.createaddrsize()`, the template files are rather simple and stable for many Fedora releases. I think I can implement this function in pure Python just fine. Just thought that it could be convenient to use it.

Do you prefer not using it/them ?